huh/db.py

```diff
@@ -81,6 +81,53 @@ def all(cls, conn: sqlite3.Connection):
         res = cur.execute(f"SELECT rowid AS id, * FROM {cls.table_name}", ())
 
         return (cls(*row) for row in res)
+    
+    @classmethod
```

Comment:
can you move these methods to the Announcement class? 
the Entry class is a base class that contains base methods that all child objects/table have (like selecting by id, getting all the entries in the sql db, etc).